zebra-state/src/service.rs

```diff
@@ -970,7 +970,7 @@ impl Service<Request> for StateService {
             Request::CommitCheckpointVerifiedBlock(finalized) => {
                 // # Consensus
```

Comment:
This comment needs a rewrite to remove all instances of "finalized block" and "non-finalised block", so I'm just going to push a commit

Reply:
See my commit 4a9b841f2d14cdd12362822da679e26b771edd56